### PR TITLE
[EventHubs] A note on how to create a batch

### PR DESCRIPTION
**Pull request by ellismg**
Add a small note and reference to the CreateBatchAsync message. I'll
admit that I initally tried to just new up an `EventDataBatch` and then
got confused when I saw it had no public constructor. Maybe this comment
saves someone from having to go to our README.md's or Samples.

### CODE REVIEW

**Comment by ellismg** [comment]
Let me know if you think this sort of thing is worthwhile, Jesse.

**Comment by serkantkaraca** [comment]
LGTM

**Review by serkantkaraca — APPROVED** [review]

File: sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducerClient.cs [hunk 1]
```diff
@@ -442,7 +442,7 @@ public virtual Task<string[]> GetPartitionIdsAsync(CancellationToken cancellatio
         ///   Sends a set of events to the associated Event Hub using a batched approach.
         /// </summary>
         ///
-        /// <param name="eventBatch">The set of event data to send.</param>
+        /// <param name="eventBatch">The set of event data to send. You can create a batch by calling CreateBatchAsync.</param>
```
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducerClient.cs]: I like the idea, but I'd like to suggest a minor tweak to better align with the tone that we've been using in the doc comments.  Maybe something like: `... A batch may be created using <see cref="CreateBatchAsync" />.`

File: sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducerClient.cs [hunk 2]
```diff
@@ -451,6 +451,7 @@ public virtual Task<string[]> GetPartitionIdsAsync(CancellationToken cancellatio
         /// <seealso cref="SendAsync(EventData, SendEventOptions, CancellationToken)" />
         /// <seealso cref="SendAsync(IEnumerable{EventData}, CancellationToken)" />
         /// <seealso cref="SendAsync(IEnumerable{EventData}, SendEventOptions, CancellationToken)" />
+        /// <seealso cref="CreateBatchAsync(CancellationToken)" />
```
Comment (jsquire) [inline sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducerClient.cs]: 👍   That's a good catch.

**Review by jsquire — APPROVED** [review]
I like the idea.   I made one small suggestion on the wording for consistency in avoiding the second person in document comments.

File: sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducerClient.cs (hunk 1 above)
Comment (ellismg) [reply sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducerClient.cs]: SGTM!

**Review by jsquire — APPROVED** [review]
LGTM.    Thanks, Matt!

**Pull request merged**